src/containers/Callers/Callers.js

```diff
@@ -139,24 +160,18 @@ class Callers extends Component {
                 method: 'GET',
                 headers: { ...authHeader(), 'Content-Type': 'application/json' },
             };
-            axios(requestOptions).then((response)=>{
-                const allCallers = response.data;
-                const callers = allCallers.map(el => {
-                  const key = el['callerId'];
-                  el.key = key;
-                  el.created = new Date(el.created.replace(/-/g, "/") + " UTC");
-                  el.lastModified = new Date(el.lastModified.replace(/-/g, "/") + " UTC");
-                  el.lastCallTimestamp = el.lastCallTimestamp ? new Date(el.lastCallTimestamp.replace(/-/g, "/") + " UTC") : null;
-                  el.lastReminderTimestamp = el.lastReminderTimestamp ? new Date(el.lastReminderTimestamp.replace(/-/g, "/") + " UTC") : null;
-                  el.contactMethodSMS = el.contactMethods.indexOf('sms') !== -1;
-                  el.contactMethodEmail = el.contactMethods.indexOf('email') !== -1;
-                  el.status = {
-                    status: callerStatus(el),
-                    monthsMissedCount: monthsMissedCount(el)
-                  };
-                  return el;
+            axios(requestOptions).then(({ data }) => {
+                // TODO: Possibly move this into reducer logic if Redux is implemented
```

Comment:
If this is only used within this container, is there a benefit to using redux for this?

Reply:
No you're totally right: If it's not being used outside of this area, there's no point to put it into redux. I put that there without looking if we used caller data elsewhere.